airflow/dags/community_dag.py

```
@@ -0,0 +1,199 @@
+
+# 커뮤니티 리스트
+# communities = ["dcinside", "bobaedream"]
+communities = ["dcinside"]  # test용
+
+# Extract Lambda 호출 Task 리스트
+extract_lambda_tasks = []
+
+# ✅ Variable.get()을 사용하여 unique_car_models 가져오기
+try:
+    unique_car_models = json.loads(Variable.get("unique_car_models"))
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)